tfhe/docs/getting_started/benchmarks/summary.md

```diff
@@ -2,6 +2,7 @@
 
 This document summarizes the timings of some homomorphic operations over 64-bit encrypted integers, depending on the hardware. More details are given for [the CPU](cpu\_benchmarks.md), [the GPU](gpu\_benchmarks.md), or [zeros-knowledge proofs](zk\_proof\_benchmarks.md).
 
+Beware that the noise used in the cryptographic parameters follows a uniform distribution, instead of a Gaussian.
```

Comment:
tweaked uniform -> TUniform not just uniform

Reply:
I modified the sentence, is it ok now?

Reply:
yep looks good to me

Reply:
don't know if it's ok with JB but this is correct